src/powerflex_logging_utilities/json_formatter.py

```diff
@@ -13,8 +13,8 @@ class JsonFormatter(jsonlogger.JsonFormatter):  # type: ignore
     - Replaces non-string keys in the log record with their __str__ representation
     """
 
-    def __init__(self, fmt: Optional[str] = DEFAULT_LOG_FORMAT, **kwargs: Any):
-        super().__init__(fmt=fmt, **kwargs)
+    def __init__(self, fmt: Optional[str] = DEFAULT_LOG_FORMAT, **kwargs: Any) -> None:
```

Comment:
Is it really necessary to return None?

Reply:
This is required for mypy's strict type checking.